### PR TITLE
Anna`s 2nd commit:

### PR DESCRIPTION
**Pull request by MedSdiri**
-added packages for seven User Stories;
-added invalid data & invalid method into Configuration Properties

Merge remote-tracking branch 'origin/master'

# Conflicts:
#	src/test/java/com/trycloud/tests/base/classToGetThis.java

### CODE REVIEW

File: configuration.properties [hunk 1]
```diff
@@ -1,8 +1,13 @@
...
 userName2=User54
 userName3=User84
 userName4=User114
 password=Userpass123
 browser=chrome
 environment=http://qa.trycloud.net/index.php/login?clear=1
+invalidPassword=yais2021
+#invalid credentials
+
+
```
Comment (ramizkazak) [inline configuration.properties:12]: Everything Looks perfect!!!

File: src/test/java/com/trycloud/tests/UserStrories/US_1_Login_feature/Main_Test.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    @Test
+    public void tempTest() {
+        LoginUtil.Login(driver, "userName1", "password");
+    }
+
+    @AfterMethod
+    public void tearDownMethod() {
+        //super.tearDownMethod();
+    }
+}
```
Comment (ramizkazak) [inline src/test/java/com/trycloud/tests/UserStrories/US_1_Login_feature/Main_Test.java:74]: Everything Looks perfect!!!

File: src/test/java/com/trycloud/tests/UserStrories/US_2_Modules_verifications/Adil.java [hunk 3]
```diff
@@ -0,0 +1,4 @@
+package com.trycloud.tests.UserStrories.US_2_Modules_verifications;
+
+public class Adil {
+}
```
Comment (ramizkazak) [inline src/test/java/com/trycloud/tests/UserStrories/US_2_Modules_verifications/Adil.java:4]: Everything Looks perfect!!!

File: src/test/java/com/trycloud/tests/UserStrories/US_2_Modules_verifications/Ahmed.java [hunk 4]
```diff
@@ -0,0 +1,4 @@
+package com.trycloud.tests.UserStrories.US_2_Modules_verifications;
+
+public class Ahmed {
+}
```
Comment (ramizkazak) [inline src/test/java/com/trycloud/tests/UserStrories/US_2_Modules_verifications/Ahmed.java:4]: Everything Looks perfect!!!

File: src/test/java/com/trycloud/tests/UserStrories/US_2_Modules_verifications/US_2_Test.java [hunk 5]
```diff
@@ -0,0 +1,4 @@
+package com.trycloud.tests.UserStrories.US_2_Modules_verifications;
+
+public class US_2_Test {
+}
```
Comment (ramizkazak) [inline src/test/java/com/trycloud/tests/UserStrories/US_2_Modules_verifications/US_2_Test.java:4]: Everything Looks perfect!!!

**Review by ramizkazak — APPROVED** [review]
Everything Looks perfect!!!

**Review by Olesea1986 — APPROVED** [review]
Good to start team !!!

**Review by ercancivi — APPROVED** [review]
it was really good group activity and it was so productive activity
Thanks

**Comment by asuman75** [comment]
hi

File: src/test/java/com/trycloud/tests/base/LoginFunctionalities.java [hunk 6]
```diff
@@ -1,57 +0,0 @@
...
-            LoginUtil.LogOut(driver);
-        }
-
-    }
-
-    @AfterMethod
-    public void tearDownMethod(){
-        super.tearDownMethod();
-    }
-}
```
Comment (ramizkazak) [inline src/test/java/com/trycloud/tests/base/LoginFunctionalities.java:57]: Everything Looks perfect!!!

**Review by GuvanchH — APPROVED** [review]
Looks Good

File: src/test/java/com/trycloud/utilities/LoginUtil.java [hunk 7]
```diff
@@ -5,15 +5,17 @@
...
+    /*public static WebDriver driver;
+    public static WebElement inputUsername = driver.findElement(By.cssSelector("input[id='user']"));
+    public static WebElement inputPassword = driver.findElement(By.xpath("//input[@id='password']"));
+    public static WebElement loginButton = driver.findElement(By.cssSelector("input[id='submit-form']"));*/
 
-    public static void Login(WebDriver driver, String key) {
-
-
+    public static void Login(WebDriver driver, String key, String password) {
         WebElement inputUsername = driver.findElement(By.cssSelector("input[id='user']"));
```
Comment (ramizkazak) [inline src/test/java/com/trycloud/utilities/LoginUtil.java:14]: Everything Looks perfect!!!

File: src/test/java/com/trycloud/utilities/LoginUtil.java [hunk 8]
```diff
@@ -31,5 +33,10 @@ public static void LogOut(WebDriver driver) {
         inputUsername.clear();
     }
 
+    public static void inputClear(WebDriver driver) {
+        WebElement inputUsername = driver.findElement(By.cssSelector("input[id='user']"));
+        inputUsername.clear();
+    }
+
```
Comment (ramizkazak) [inline src/test/java/com/trycloud/utilities/LoginUtil.java:40]: Everything Looks perfect!!!

Comment (ramizkazak) [inline src/test/java/com/trycloud/utilities/LoginUtil.java:40]: Everything Looks perfect!!!

File: src/test/java/com/trycloud/utilities/LoginUtil.java [hunk 9]
```diff
@@ -5,15 +5,17 @@
...
-    public static void Login(WebDriver driver, String key) {
-
-
+    public static void Login(WebDriver driver, String key, String password) {
         WebElement inputUsername = driver.findElement(By.cssSelector("input[id='user']"));
         inputUsername.sendKeys(ConfigurationReader.getProperty(key));
 
         WebElement inputPassword = driver.findElement(By.xpath("//input[@id='password']"));
-        inputPassword.sendKeys(ConfigurationReader.getProperty("password"));
+        inputPassword.sendKeys(ConfigurationReader.getProperty(password));
```
Comment (ramizkazak) [inline src/test/java/com/trycloud/utilities/LoginUtil.java:19]: Everything Looks perfect!!!

File: src/test/java/com/trycloud/utilities/LoginUtil.java (hunk 8 above)
Comment (ramizkazak) [inline src/test/java/com/trycloud/utilities/LoginUtil.java:40]: Everything Looks perfect!!!

File: src/test/java/com/trycloud/utilities/LoginUtil.java [hunk 10]
```diff
@@ -5,15 +5,17 @@
...
 public class LoginUtil {
+    /*public static WebDriver driver;
+    public static WebElement inputUsername = driver.findElement(By.cssSelector("input[id='user']"));
+    public static WebElement inputPassword = driver.findElement(By.xpath("//input[@id='password']"));
+    public static WebElement loginButton = driver.findElement(By.cssSelector("input[id='submit-form']"));*/
 
-    public static void Login(WebDriver driver, String key) {
-
-
+    public static void Login(WebDriver driver, String key, String password) {
```
Comment (ramizkazak) [inline src/test/java/com/trycloud/utilities/LoginUtil.java:13]: Everything Looks perfect!!!

Comment (ramizkazak) [inline src/test/java/com/trycloud/utilities/LoginUtil.java:13]: Everything Looks perfect!!!

File: src/test/java/com/trycloud/tests/base/LoginFunctionalities.java (hunk 6 above)
Comment (ramizkazak) [inline src/test/java/com/trycloud/tests/base/LoginFunctionalities.java:57]: Everything Looks perfect!!!

**Review by ramizkazak — COMMENTED** [review]
Everything Looks perfect!!!

**Pull request closed without merging**